validators/types_unix.go

```diff
@@ -40,11 +40,12 @@ var DefaultSysSpec = SysSpec{
 			{Name: "IPC_NS"},
 			{Name: "UTS_NS"},
 			{Name: "CGROUPS"},
-			{Name: "CGROUP_CPUACCT"},
+			{Name: "CGROUP_BPF"},     // cgroups v2
+			{Name: "CGROUP_CPUACCT"}, // cgroups v1 cpuacct
 			{Name: "CGROUP_DEVICE"},
 			{Name: "CGROUP_FREEZER"},
```

Comment:
CONFIG_CGROUP_FREEZER is for cgroups v1 freezer.
According to the below, we may disable CONFIG_CGROUP_FREEZER when we
use cgroups v2 freezer.
https://github.com/torvalds/linux/blob/v6.12-rc1/init/Kconfig#L1132